openc3-redis/Dockerfile

```diff
@@ -16,28 +15,30 @@ ENV REQUESTS_CA_BUNDLE=/devel/cacert.pem
 ENV NODE_EXTRA_CA_CERTS=/devel/cacert.pem
 
 USER root
+# Modify the redis user and group to be 1001
+# The default redis container uses 999
+# See https://github.com/docker-library/redis/blob/master/7.2/alpine/Dockerfile
+RUN apk update && apk add shadow && \
+  usermod -u 1001 redis && \
+  groupmod -g 1001 redis
 
 # Update packages to eliminate CVEs if we're on docker.io (not ironbank)
 RUN if [[ $OPENC3_DEPENDENCY_REGISTRY == 'docker.io' ]]; then \
-  # add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
-  set -eux; \
-	groupadd -r -g 1001 redis; \
-	useradd -r -g redis -u 1001 redis; \
-  apt update && apt upgrade -y; \
+  apk upgrade; \
   fi
 
 RUN mkdir /config
 COPY redis.conf /config/.
 COPY redis_ephemeral.conf /config/.
 COPY users.acl /config/.
-COPY --chmod=0755 ./docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
```

Comment:
This is already part of the alpine docker image so I removed it (don't think it was being used anyway)

Reply:
I modified the default one for a reason.  Probably needs to be put back.